src/style.css

```diff
@@ -0,0 +1,25 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  width: 96px;
```

Comment:
Try not to hardcode width and remove `justify-content: space-between;` rule and set correct margin-right (4px) for each star except last star in a row and re-run your tests

Reply:
i remake and tests are corrects, but can you explain pls, why idea to hardcode witdth and add justify-content: space-between; is bad?

Reply:
good practise to allow content make size of container
in this case it is better solution. for example if will need to add more stars, we can add and will not change styles.